teekam_singh/PostgresAssignment/src/services/class.ts

```diff
@@ -0,0 +1,45 @@
+import { query, setpath, start } from "../db/database";
+
+export async function createClass(id: string, name: string) {
+
+    try {
+        await start();
```

Comment:
no need to start for every request can you do it once before you do app.listen , same for setpath